### ISSUE TITLE
Use custom fields

### ISSUE THREAD

**Issue by mr-manuel** [MCOfficer/bitwarden-autotype#141]
Hi,
thanks for this great project!

Would it be possible to also use the custom fields of a Bitwarden entry in the autotype sequence?

**Comment by KnightTim** [comment]
This looks to be a dup of #49 which I submitted over a year ago.  I think this project maybe abandoned at this point. 

Personally, I have switched over to using https://github.com/anonymous1184/bitwarden-autotype which is very similar but more feature complete, IMO.

**Comment by MCOfficer** [comment]
Yeah - no matter my intentions, let's make it official: I don't see myself coming back to this anytime soon, sorry.

8778fae401a0de7969195dfc2c72cd7f8cd8ae39
